4sale.com/templates/post-ad.html

```diff
@@ -232,11 +232,21 @@ <h2>Post an Ad</h2>
+ 	new Awesomplete(input, {list: localityList});
 
  </script>
+ <script>
+	 function validateForm(form)
+	 {
+		 var x= document.getElementById("locality").value;
+		 if(localityList.indexOf(x) < 0) {
```

Comment:
@chakradhaarrv who gave idea for this hmmmmmmmmmm

Reply:
lol